src/knx_facade.cpp

```diff
@@ -42,14 +42,19 @@
     #endif
 #elif __linux__

```

Comment:
just threat linux as all other non arduino platforms here. There is no reason di differentiate them, right?

Reply:
changed it.